addons/hr_timesheet/views/hr_employee_views.xml

```diff
@@ -38,23 +38,6 @@
         </field>
     </record>
 
-    <record id="hr_department_view_kanban" model="ir.ui.view">
```

Comment:
this is very strange :eyes: 
https://github.com/odoo/odoo/blame/master/addons/hr_timesheet/views/hr_department_views.xml

https://github.com/odoo/odoo/blame/master/addons/hr_timesheet/views/hr_employee_views.xml

and those lines, blame to the same commit as well :exploding_head: 

Probably something went wrong 

https://github.com/odoo/odoo/commit/d01399589f83741a135197052c102365fa8a43b2
here, file was renamed.. 
and this view was added, that was already existing

so, we can remove it